specificatie/unresolved/openapi.yaml

```diff
@@ -0,0 +1,2136 @@
+        'default':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Bevragingen-ingeschreven-personen/master/api-specificatie/common.yaml#/components/responses/default"
+      tags: 
+      - Beslagen
+  /beslagen:
+    get:
+      operationId: GetBeslagen
+      description: "Het ophalen van beslagen die door een specifieke beslaglegger zijn gelegd."
+      parameters: 
+        - in: query
+          name: burgerservicenummer
+          description: "Het burgerservicenummer van de ingeschreven persoon."
+          required: false
+          schema:
+            type: string
+            maxLength: 9
+        - in: query
+          name: kadastraleidentificatie
+          description: "De kadastrale identificatie van een persoon of een rechtspersoon die bij het kadaster geregistreerd is."
+          required: false
+          schema:
+            type: string
+            maxLength: 16
+        - in: query
+          name: type
+          description: "Het type betrokkene die beslag heeft gelegd op een Kadastraal Onroerende Zaak."
+          required: true
+          schema: 
+            type: string
+            maxLength: 15
+      responses:
+        '200':
+          description: "Zoekactie geslaagd"
+          headers:
+            api-version:
+              $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Bevragingen-ingeschreven-personen/master/api-specificatie/common.yaml#/components/headers/api_version"
+            warning:
+              $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Bevragingen-ingeschreven-personen/master/api-specificatie/common.yaml#/components/headers/warning"
+            X-Rate-Limit-Limit:
+              $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Bevragingen-ingeschreven-personen/master/api-specificatie/common.yaml#/components/headers/X_Rate_Limit_Limit"
+            X-Rate-Limit-Remaining:
+              $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Bevragingen-ingeschreven-personen/master/api-specificatie/common.yaml#/components/headers/X_Rate_Limit_Remaining"
+            X-Rate-Limit-Reset:
+              $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Bevragingen-ingeschreven-personen/master/api-specificatie/common.yaml#/components/headers/X_Rate_Limit_Reset"
+          content:
+            application/hal+json:
+              schema:
+                $ref: "#/components/schemas/BeslagHalCollectie"
+        '400':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Bevragingen-ingeschreven-personen/master/api-specificatie/common.yaml#/components/responses/400"
+        '401':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Bevragingen-ingeschreven-personen/master/api-specificatie/common.yaml#/components/responses/401"
+        '403':
+          $ref: "ht
```

Comment:
Wat is de verwachting bij het gebruiken van de fields parameter in combinatie met required? Krijgt je deze required field ook al geeft je dit niet op met de fields parameter? Als dat niet het geval is, dan voldoet de json data niet aan de specificatie

Reply:
Goede vraag. Morgen even in de groep neerleggen. Bij het kadaster hebben sommige gegevens alleen betekenis in samenhang met andere gegevens. Als die niet in samenhang gelkeverd worden wordt er ook geen deel geleverd. Ik weet echter niet of we dat met required kunnen afdekken., Ik denk het eigenlijk niet. 
Morgen bespreken of we (analoog aan BRP) voor bevragen alles optioneel maken ? Als voorschot op die discussie doe ik dat.